app/experimenter/experiments/api/v5/types.py

```diff
@@ -186,9 +185,6 @@ def resolve_can_review(self, info):
     def resolve_review_request(self, info):
         return self.changes.latest_review_request()
 
-    def resolve_review_approval(self, info):
-        return self.changes.latest_review_approval()
-
```

Comment:
Didn't we just add this?  Why are we taking it out?

Reply:
Removing the reviewApproval changelog event exposed in GQL, since it became unnecessary after another rework of ChangeApprovalOperations to use publishStatus

Reply:
Oh okay that makes sense 👍 Bye code!